app/src/main/java/io/bloco/circles/domain/NewIdentity.kt

```diff
@@ -16,17 +16,17 @@ class NewIdentity
     private val uploadProfile: UploadProfile,
     private val uploadWallet: UploadWallet,
 ) {
-    suspend fun create(username: String): Result<IdentityModel> = try {
+    suspend fun create(username: String?): Result<IdentityModel> = try {
         var identities = identityRepository.observe().first()
         val keys = identityKeys.new()
         val btcAddress = keys.toBtcAddress()
 
-        val newIdentity = generateIdentity.generate(btcAddress, username)
+        val newIdentity = generateIdentity.generate(btcAddress, username ?: btcAddress)
```

Comment:
Couldn't this create some issues in the future, mixing the username with the address? Shouldn't we just allow null usernames on the IdentityModel? Or that's how we need it to be for sending data?